constants/const.go

```diff
@@ -0,0 +1,44 @@
+package constants
+
+const (
+	OSExitSuccess = 0
+	OSExitError   = -1
+)
+
+type OutMode int64
+
+// Const values for configuration mode field.
+const (
+	_ OutMode = iota
+	PrintAll
+	PrintSubsys
+	PrintSubsysWs
+	PrintTargeted
+	OutModeLast
+)
+
+// Const values for output type.
+const (
+	InvalidOutput int = iota
+	GraphOnly
+	JsonOutputPlain
+	JsonOutputB64
+	JsonOutputGZB64
+)
+
+// Configuration defaults.
+const (
+	DefaultMode       = PrintSubsys
+	DefaultOutputType = "graphOnly"
+	DefaultSymbol     = ""
+	DefaultMaxDepth   = 0
+	DefaultDBDriver   = "postgres"
+	DefaultDBInstance = 1
+	DefaultDBDSN      = ""
+)
```

Comment:
What purpose is served by having default values that can't work, like "" for DefaultSymbol and DefaultDBDSN? Reading  initFlagSet it looks like there are defaults for everything, but actually these two have to be defined by the user (command line arguments or config file). Should we remove these constants and just set nil as default value, or something like that? I'm ot experienced enough with pflag to know what the canonical way is.

Reply:
This is kind of a leftover from the "reasonable default value for everything" approach that Alessandro argues for. Right now, they indeed don't serve much of a purpose. 
I think it depends on if we want to always have the user to set those (then we can remove the constants) or if we want to have a reasonable default value that the program jumps to if it doesn't receive it (then we can use the constants).

Reply:
I'm very much in favour of reasonable default values, if they exist. I doubt that for these two they do.

Reply:
Removing the redundant constants in my next commit, resolving this conversation